code/ui/blocks/src/components/DocsPage.stories.tsx

```diff
@@ -95,3 +96,28 @@ export const Markdown = () => (
     <Source {...source.JSX.args} />
   </DocsPageWrapper>
 );
+
+export const Html = {
+  name: 'HTML',
+  render: () => (
+    <DocsPageWrapper>
+      <h1>Heading 1</h1>
+      <h2>Heading 2</h2>
+      <a>A tag</a>
+      <pre>pre tag</pre>
+      <div>
+        <div>Div</div>
+        <a>Nested A tag</a>
+      </div>
+      <div className="sb-unstyled" style={{ border: '2px solid red' }}>
```

Comment:
Why not use the `Unstyled` component here?

Reply:
You're right, I did this before I built the component.